The Globe and Mail.js

```diff
@@ -155,7 +171,11 @@ var testCases = [
 					"Rob Ford",
 					"leadership"
 				],
-				"notes": [],
+				"notes": [
+					"http://www.theglobeandmail.com/news/toronto/doug-ford-says-ontario-pc-leadership-bid-is-on-the-table/article21358827/",
+					" http://www.theglobeandmail.com/news/toronto/uniting-a-divided-toronto-will-be-a-key-task-for-torys-transition-team/article21359883/",
+					" http://www.theglobeandmail.com/news/toronto/a-chuckling-ford-on-his-mayoralty-it-will-definitely-be-remembered/article21414526/"
+				],
```

Comment:
The `notes` contains an array of objects with a parameter `note` where the content is saved. Actually, I would suggest to just create one note with all urls. Compare for example the [translator for beck-online](https://github.com/zotero/translators/blob/master/beck-online.js).